cloud: add integration with GPT (#15424)
pingcap/docs: tidb-cloud/data-service-manage-data-app.md

For more information, see [Deploy automatically with GitHub](/tidb-cloud/data-service-manage-github-connection.md).

> ### Integrate your Data App with GPT

Author: ```suggestion
### Integrate your Data App with GPTs
```

> You can integrate your Data App with GPT to enhance your applications with intelligent capabilities.

Author: ```suggestion
You can integrate your Data App with GPTs to enhance your applications with intelligent capabilities.
```

> You can integrate your Data App with GPTs to enhance your applications with intelligent capabilities.

Reviewer: ```suggestion
You can integrate your Data App with [GPTs](https://openai.com/blog/introducing-gpts) to enhance your applications with intelligent capabilities.
```

> To integrate your Data App with GPT, perform the following steps:

Author: ```suggestion
To integrate your Data App with GPTs, perform the following steps:
```

1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
2. In the left pane, locate your target Data App and click the name of your target Data App to view its details.

> 3. In the **Integration with GPT** area, click **Get Configuration**.

Author: ```suggestion
3. In the **Integration with GPTs** area, click **Get Configuration**.
```

4. In the displayed dialog box, you can see the following fields:

    - **API Specification URL**: copy the URL of the OpenAPI Specification of your Data App. For more information, see [Use the OpenAPI Specification](#use-the-openapi-specification).

>     - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).

Reviewer: ```suggestion
    - **API Key**: enter the API key of your Data App. If you do not have an API key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
```

>     - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. Click **Encode** to encode the API Key. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).

Reviewer 2: [Click **Encode**] 这个按钮去掉了，输入apikey之后在下面会自动生成编码之后的内容

Author: Create API Key 按钮也删掉了是吗？

>     - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. Click **Encode** to encode the API Key. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).

Author: ```suggestion
    - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
```

>     - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. Click **Encode** to encode the API Key. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).

Author: ```suggestion
    - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
```

>     - **API Key Encoded**: copy the base64 encoded string equivalent of the API Key you have provided.

Reviewer: ```suggestion
    - **API Key Encoded**: copy the base64 encoded string equivalent to the API key you have provided.

```

> 5. Copy the API Specification URL and the encoded API Key, and use them in the GPT configuration.

Reviewer: 第 4 步里已经告诉要 copy 了，这里要再次 copy 吗

Author: 改成下面这样了。PTAL

> 5. Copy the API Specification URL and the encoded API Key, and use them in the GPT configuration.

Author: ```suggestion
5. Now you can use the coped API Specification URL and the encoded API key in the GPT configuration.
```
